src/boardlib/__main__.py

```diff
@@ -32,8 +33,9 @@ def logbook_entries(board, username, password, grade_type="font"):
         raise ValueError(f"Unknown board {board}")
 
 
-def write_entries(output_file, entries, no_headers=False):
-    writer = csv.DictWriter(output_file, LOGBOOK_FIELDS)
+def write_entries(output_file_path, entries, no_headers=False):
+    with open(output_file_path, mode='w', newline='', encoding='utf-8') as output_file:
```

Comment:
Should be able to remove this change with the suggestions I made to avoid the funky-looking double sys.stdout open. Let me know if it solves your issue. I think my `locale` default is to encode utf-8 which is why I didn't see these issues.

Reply:
Oh that makes sense.